packages/components/src/form/inputs/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,93 @@
+import { cns } from '@stcland/utils'
+import { useFormContext } from 'react-hook-form'
+import { nullFormContext } from '../Form'
+import FormLabel from '../FormLabel'
+
+//*****************************************************************************
+// Interface
+//*****************************************************************************
+type RadioButtonShape = {
+  label: string;
+  value: string;
+};
+interface Props {
+  name: string;
+  label?: string;
+  direction?: 'vertical' | 'horizontal';
+  defaultChecked?: string;
+  radioButtons: RadioButtonShape[];
+  readOnly?: boolean;
+  required: boolean;
+  disabled?: boolean;
+  hidden?: boolean;
+  className?: string;
+}
+
+const toId = function (name: string, radioButtonValue: string): string {
+  return name + '.' + radioButtonValue
+}
+
+const RadioButtonGroup = function ({
+  name,
+  label,
+  radioButtons,
+  disabled = false,
+  hidden = false,
+  direction = 'horizontal',
+  defaultChecked = '',
+  required = false,
+  readOnly = false,
+  className,
+}: Props) {
+  if (hidden) return null
+
+  const { register } = useFormContext() || nullFormContext
+  const registerRequired = {
+    value: required,
+    message: label + ' is required',
+  }
+  const flexDir = direction === 'horizontal' ? 'flex-row' : 'flex-col'
+
+  const cn = {
+    root: cns('mb-2 flex gap-4', className),
+    buttonContainer: cns('flex', flexDir),
+    label: 'text-white/50',
+    radioContainer: 'flex gap-2',
+    radioButton: (row: number) =>
+      cns(
+        'accent-app-primary-500',
+        direction === 'horizontal' && row !== 0 && 'flex-3',
+      ),
+  }
```

Comment:
I think we should create a `RadioButtonGroupStyles`, and also pass a `customStyles` prop... so we can allow dynamic styles from consumer perspective.